### PR TITLE
Support test filtering

### PR DESCRIPTION
**Pull request by andreasohlund**
Tested with https://github.com/Particular/ServiceControl/pull/3417

### CODE REVIEW

File: run-tests.ps1 [hunk 1]
```diff
@@ -49,9 +49,10 @@ $projects | ForEach-Object {
 $testProjects = $testProjects.GetEnumerator() | Sort-Object Name
 $testFrameworks = $testFrameworks.GetEnumerator() | Sort-Object
 $reportWarnings = 'false'
+$filter = ''
 
-if ($Env:REPORT_WARNINGS -eq 'true') {
-    $reportWarnings = 'true'
```
Comment (DavidBoike) [inline run-tests.ps1]: Don't think you meant to delete that?

**Review by DavidBoike — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -58,6 +58,16 @@ By default, `dotnet test` uses `x64` as the target platform. This can be overrid
           target-platform: x86
 ```
 
+To filter tests use the filter argument together with the [dotnet test filter syntax(https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests)
+
+```yaml
+    steps:
+      - name: Run tests
+        uses: Particular/run-tests-action@v1.3.0
```
Comment (DavidBoike) [inline README.md]: Right before you merge and tag the new version, change this (and all the other versions) to the new version, I assume `v1.4.0`.

File: run-tests.ps1 (hunk 1 above)
Comment (andreasohlund) [reply run-tests.ps1]: Ugh, good catch, fixed

File: README.md (hunk 2 above)
Comment (andreasohlund) [reply README.md]: Made it 1.4.0 in this PR

**Review by DavidBoike — APPROVED** [review]

**Pull request merged**